lncrawl/binders/epub.py

```diff
@@ -74,6 +80,9 @@ def bind_epub_book(
             <h3>{novel_author}</h3>
         </div>
         <img class="cover" src="{COVER_IMAGE_NAME}">
+        <div class="synopsis">
```

Comment:
instead of putting the synopsis with the cover page, it would be better to make a separate page for it